ci/hydra_spec.json

```diff
@@ -0,0 +1,29 @@
+{
+  "enabled": 1,
+  "hidden": false,
+  "description": "nix-overlays tests",
+  "nixexprinput": "src",
+  "nixexprpath": "ci/hydra_jobs.nix",
+  "checkinterval": 300,
+  "schedulingshares": 100,
+  "enableemail": false,
+  "emailoverride": "",
+  "keepnr": 10,
+  "inputs": {
+    "nixpkgs": {
```

Comment:
Are you sure we need this? Supposedly we'll be building against a specific hash of nixpkgs